lib/webpacker/manifest.rb

```diff
@@ -0,0 +1,53 @@
+# Singleton registry for accessing the packs path using generated manifest.
+# This allows javascript_pack_tag or stylesheet_pack_tag to take a reference to,
+# say, "calendar.js" or "calendar.css" and turn it into "/packs/calendar.js" or
+# "/packs/calendar.css" in development. In production mode, it returns digested
+# files, # "/packs/calendar-1016838bab065ae1e314.js" and
+# "/packs/calendar-1016838bab065ae1e314.css" for long-term caching
+
+class Webpacker::Manifest
```

Comment:
Previously, it was used for looking up digests on production and it made, but now this is used in all environments to lookup assets using manifest file, not just for digesting. What do you think?